game-core/src/main/java/games/strategy/engine/framework/lookandfeel/LookAndFeel.java

```diff
@@ -78,6 +79,18 @@ public static String getDefaultLookAndFeelClassName() {
             .orElseGet(UIManager::getSystemLookAndFeelClassName);
   }
 
+  public static boolean isCurrentLookAndFeelDark() {
+    final Color background = UIManager.getColor("Panel.background");
+    return background != null && isColorDark(background);
+  }
+
+  public static boolean isColorDark(final Color color) {
```

Comment:
Would you mind adding some unit testing for this function?

Reply:
Done.